package.json

```diff
@@ -26,7 +23,7 @@
     "docs": "yarn analyze && web-dev-server --node-resolve --open",
     "lint": "npm-run-all --parallel lint:*",
     "lint:css": "stylelint packages/**/src/*.js packages/**/theme/**/*-styles.js",
-    "lint:js": "eslint *.js scripts packages",
+    "lint:js": "eslint --ext .js,.ts scripts packages",
```

Comment:
Just to double check this change. It looks like the original specified 3 globs:
- *.js
- scripts
- packages

The change sets the extensions to look for, but then only lists two globs:
- scripts
- packages

Am I seeing this right?  It would mean that files in the root folder are not linted anymore.

Reply:
Good catch. Fixed. Thanks!